pkg/notify/notify_test.go

```diff
@@ -0,0 +1,428 @@
+package notify
+
+import (
+	"fmt"
+	"os"
+	"sort"
+	"testing"
+	"time"
+
+	"github.com/trivago/tgo/tcontainer"
+
+	"github.com/andygrunwald/go-jira"
+	"github.com/free/jiralert/pkg/alertmanager"
+	"github.com/free/jiralert/pkg/config"
+	"github.com/free/jiralert/pkg/template"
+	"github.com/go-kit/kit/log"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/require"
+)
+
+func TestToGroupTicketLabel(t *testing.T) {
+	require.Equal(t, `ALERT{C="d",a="B"}`, toGroupTicketLabel(alertmanager.KV{"a": "B", "C": "d"}))
+}
+
+type mockedJira struct {
```

Comment:
Good point, it's a fake. My constructor even has that name ;p 

I don't use any magic mocking because, well -> I wanted to do fake than mock more and don't want to introduce new deps if not neccesary. Are you happy with renaming to `fake` and leaving it like is? We can improve later if needed.

Reply:
I wanted fake as it's just easier to follow on test cases as they mimick natural flow of alerts/tickets

Reply:
TBH, whenever someone writes tests I'm just glad they did it and leave it at that. So if you find it easy to follow this way I won't object. Just name it `fakeJira` and the constructor `newFakeJira` and let it be.

And thanks, BTW.